ksql-functional-tests/src/test/resources/query-validation-tests/insert-into.json

```diff
@@ -50,8 +50,8 @@
       "name": "table",
       "statements": [
         "CREATE STREAM SOURCE (d1 VARCHAR) WITH (kafka_topic='SOURCE', value_format='DELIMITED');",
-        "CREATE TABLE OUTPUT (d1 VARCHAR, COUNT INT) WITH (kafka_topic='OUTPUT', value_format='DELIMITED', key='d1');",
-        "INSERT INTO OUTPUT SELECT d1, COUNT() FROM SOURCE GROUP BY d1;"
+        "CREATE TABLE OUTPUT (d1 VARCHAR, COUNT BIGINT) WITH (kafka_topic='OUTPUT', value_format='DELIMITED', key='d1');",
```

Comment:
Not a huge deal since the error message is being asserted, but it's a more specific setup for the test. Previously, the statement had 3 things wrong with it:

- the schema for `COUNT` in the `OUTPUT` was `INT`, but the `INSERT INTO`, was producing an aggregation result with type `BIGINT` (schema mismatch)
- the result of `COUNT()` was the default aggregation result name, not `COUNT` (schema mismatch)
- the sink is a table

This way the statements are correct except for the result being a table (which is what's being tested).